package/MDAnalysis/core/universe.py

```diff
@@ -74,6 +74,8 @@
 .. autofunction:: Merge
 
 """
+from cStringIO import InputType
```

Comment:
Also, import from the standard library should be kept together *after* the compatibility import (i.e. `__future__` and `six`).